Plugins/Statistics/GitStatistics/FormGitStatistics.cs

```diff
@@ -82,7 +83,7 @@ public FormGitStatistics(IGitModule module, string codeFilePattern, bool countSu
             InitializeComplete();
         }
 
-        private void FormGitStatisticsSizeChanged(object sender, EventArgs e)
+        private void FormGitStatisticsSizeChanged(object? sender, EventArgs? e)
```

Comment:
I don't think this is correct. Handlers generally have the following annotation: `(object? sender, EventArgs e)`

Reply:
Agree this looks wrong. As before, I've just been annotating the current reality and fixing things up only when needed. I'll see if this is ready to change.